### ISSUE TITLE
Can make share pre-traied model?

### ISSUE THREAD

**Issue by faranaziz** [Tinglok/CVC#7]
Thanks you

**Comment by Tinglok** [comment]
Sorry for the delay, the pre-trained model has been deleted unexpectedly. I will train a new one and upload it in the future.
If you are in a hurry, you can train one by yourself follow the instructions (the default parameters are all set).
Feel free to reach out if you have any further questions.

**Comment by Tinglok** [comment]
Pre-trained models are now available in this [URL](https://drive.google.com/drive/folders/1B21i1FoHQaK3KZ9CDjyUK4qDZdj4VfiR?usp=sharing).

**Comment by Tinglok** [comment]
BTW, I didn't train long for the pre-trained model. Feel free to contribute if you have better result.

**Comment by AlexFuster** [comment]
yeah, I see you trained for 19 epochs. I can push the model from there if you think that it would improve

**Comment by AlexFuster** [comment]
In the paper I can see that you trained the model for 1000 epochs. Is that correct? I mean, you said that 145 epochs is crazy, that's why I find it strange

**Comment by Tinglok** [comment]
Yeah, but that's only applicable for the one-to-one model. For many-to-one model, since the number of trainA is large, the training will converge quicker and be more stable.

**Comment by Tinglok** [comment]
I wonder if the current pre-trained model works well for you? If not, it's that possible that there's something wrong with your PWG vocoder?

**Comment by AlexFuster** [comment]
It is working and giving intelligible results. Although it has style leaking in VC across gender. Specially compared with the results reported in the website. I'll do further tests and tell you.

**Comment by Tinglok** [comment]
Thanks! Another trick I used before is that I ruled out the utterances shorter than 2-second instead of zero-padding, though I didn't know if that matters.
